### PR TITLE
Add styles to section channel

### PR DESCRIPTION
**Pull request by Gatts1**
- Create bottom channel section where it shows user state.
- Customize Input to create a new channel.
-Customize channel list.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -80,14 +80,55 @@
...
+              <path
+                fill="currentColor"
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M5.88657 21C5.57547 21 5.3399 20.7189 5.39427 20.4126L6.00001 17H2.59511C2.28449 17 2.04905 16.7198 2.10259 16.4138L2.27759 15.4138C2.31946 15.1746 2.52722 15 2.77011 15H6.35001L7.41001 9H4.00511C3.69449 9 3.45905 8.71977 3.51259 8.41381L3.68759 7.41381C3.72946 7.17456 3.93722 7 4.18011 7H7.76001L8.39677 3.41262C8.43914 3.17391 8.64664 3 8.88907 3H9.87344C10.1845 3 10.4201 3.28107 10.3657 3.58738L9.76001 7H15.76L16.3968 3.41262C16.4391 3.17391 16.6466 3 16.8891 3H17.8734C18.1845 3 18.4201 3.28107 18.3657 3.58738L17.76 7H21.1649C21.4755 7 21.711 7.28023 21.6574 7.58619L21.4824 8.58619C21.4406 8.82544 21.2328 9 20.9899 9H17.41L16.35 15H19.7549C20.0655 15 20.301 15.2802 20.2474 15.5862L20.0724 16.5862C20.0306 16.8254 19.8228 17 19.5799 17H16L15.3632 20.5874C15.3209 20.8261 15.1134 21 14.8709 21H13.8866C13.5755 21 13.3399 20.7189 13.3943 20.4126L14 17H8.00001L7.36325 20.5874C7.32088 20.8261 7.11337 21 6.87094 21H5.88657ZM9.41045 9L8.35045 15H14.3504L15.4104 9H9.41045Z"
+              ></path>
+            </svg>
```
Comment (Linzeur) [inline public/index.html]: You could define this in the section where are all svg tag, and here you call to it for its id

File: public/index.html [hunk 2]
```diff
@@ -80,14 +80,55 @@
       </div>
       <div>
         <ul id="channels_list">
-          <li class="text_channels">Example channel name</li>
+          <li class="text_channels">
+            <svg width="24" height="24" viewBox="0 0 24 24" class="icon-1_QxNX">
```
Comment (paulotijero) [inline public/index.html]: This class `icon-1_QxNX` don't exist in your `style.css`

File: public/index.html [hunk 3]
```diff
@@ -80,14 +80,55 @@
...
+                clip-rule="evenodd"
+                d="M5.88657 21C5.57547 21 5.3399 20.7189 5.39427 20.4126L6.00001 17H2.59511C2.28449 17 2.04905 16.7198 2.10259 16.4138L2.27759 15.4138C2.31946 15.1746 2.52722 15 2.77011 15H6.35001L7.41001 9H4.00511C3.69449 9 3.45905 8.71977 3.51259 8.41381L3.68759 7.41381C3.72946 7.17456 3.93722 7 4.18011 7H7.76001L8.39677 3.41262C8.43914 3.17391 8.64664 3 8.88907 3H9.87344C10.1845 3 10.4201 3.28107 10.3657 3.58738L9.76001 7H15.76L16.3968 3.41262C16.4391 3.17391 16.6466 3 16.8891 3H17.8734C18.1845 3 18.4201 3.28107 18.3657 3.58738L17.76 7H21.1649C21.4755 7 21.711 7.28023 21.6574 7.58619L21.4824 8.58619C21.4406 8.82544 21.2328 9 20.9899 9H17.41L16.35 15H19.7549C20.0655 15 20.301 15.2802 20.2474 15.5862L20.0724 16.5862C20.0306 16.8254 19.8228 17 19.5799 17H16L15.3632 20.5874C15.3209 20.8261 15.1134 21 14.8709 21H13.8866C13.5755 21 13.3399 20.7189 13.3943 20.4126L14 17H8.00001L7.36325 20.5874C7.32088 20.8261 7.11337 21 6.87094 21H5.88657ZM9.41045 9L8.35045 15H14.3504L15.4104 9H9.41045Z"
+              ></path>
+            </svg>
+            <span class="each_channel">Example channel name</span>
+          </li>
         </ul>
```
Comment (diegotc86) [inline public/index.html]: This is the base `li` we should use when we generate new elements in javascript. Thanks

File: public/index.html (hunk 2 above)
Comment (Gatts1) [reply public/index.html]: Yes, it is used in css
```
.icon-1_QxNX {
  color: #8e9297;
  display: block;
}
```

File: public/index.html [hunk 4]
```diff
@@ -80,14 +80,55 @@
...
+            <button class="icon_button_micro">
+              <img src="assets/img/micro.jpg" alt="" height="80%" width="80%" />
+            </button>
+            <button
+              class="icon_button_audio">
+              <img src="assets/img/audio.jpeg" alt="" height="80%" width="80%">
+            </button>
+            <button
+              class="icon_button_settings">
+              <img src="assets/img/settings.png" alt="" height="80%" width="80%">
```
Comment (paulotijero) [inline public/index.html]: Could use the `svg` what is on the official website:
svg
 You could create an issue to refactor this.

Comment (diegotc86) [reply public/index.html]: Nice comment

File: public/index.html (hunk 2 above)
Comment (paulotijero) [reply public/index.html]: Oh, sorry, I don't see it, maybe you can change the name that this class :thinking:

File: public/index.html (hunk 4 above)
Comment (Linzeur) [reply public/index.html]: Good gif of explanation

**Pull request merged**